ext/app/electron/main.ts

```diff
@@ -28,6 +28,8 @@ setDefaultEnv('GRIST_ORG_IN_PATH', 'true');
 setDefaultEnv('APP_UNTRUSTED_URL', 'http://plugins.invalid');
 setDefaultEnv('GRIST_HIDE_UI_ELEMENTS', 'helpCenter,billing,templates,multiSite,multiAccounts');
 setDefaultEnv('GRIST_ELECTRON_AUTH', 'strict');
+setDefaultEnv('GRIST_WIDGET_LIST_URL',
+  'https://github.com/gristlabs/grist-widget/releases/download/latest/manifest.json');
```

Comment:
How about centralizing this constant in app/common/gristUrls.ts:commonUrls?
Apart from that, seems good.

Reply:
I updated the PR in grist-core, but here, I can't do that without updating the grist-core submodule.